payments_service/braintree/forms.py

```diff
@@ -73,3 +61,18 @@ def user_owns_resource(self, uri, lookup):
             log.debug('{cls}: catching {e.__class__.__name__}: {e}'
                       .format(cls=self.__class__.__name__, e=exc))
             return False
+
+
+class ChangeSubscriptionPayMethodForm(ManageSubscriptionForm):
+    new_pay_method_uri = forms.CharField(max_length=255)
+
+    def clean_new_pay_method_uri(self):
+        uri = self.cleaned_data['new_pay_method_uri']
+        if not self.user_owns_resource(
+            uri,
+            {'braintree_buyer__buyer__uuid': self.user.uuid},
+        ):
```

Comment:
if I use a hanging indent it has to be indented twice since it's an if statement. Dunno, both look pretty weird.